api/src/test/java/org/openmrs/api/EncounterServiceTest.java

```diff
@@ -425,7 +425,7 @@ private Encounter buildEncounter() {
 		enc.setLocation(Context.getLocationService().getLocation(1));
 		enc.setEncounterType(Context.getEncounterService().getEncounterType(1));
 		enc.setEncounterDatetime(new Date());
-		enc.setPatient(Context.getPatientService().getPatient(3));
+		enc.setPatient(Context.getPatientService().getPatient(2));
```

Comment:
Am trying to fix some tests that are failing on this i have started with saveEncounter_shouldNotOverwriteObsAndOrdersCreatorOrDateCreated()  in EncounterServiceTest.java.According to my changes,For this method,i expect it to pass on pointing to the same patient for both the obs and the encounter, but this still breaks even on pointing to the same patient for both the obs and encounter for this method.Any suggestions to get this fixed cc @dkayiwa @ibacher  @mks-d

Reply:
I thought you were to start by adding the failing test in the ticket description, then work towards making it pass.

Reply:
Am having a couple of failing tests for this after my changes and i am solving one by one as per my comments above